lib/tizenCertificateManager.js

```diff
@@ -6,15 +6,7 @@ const crypto = require('crypto');
 const cryptUtil = require('./cryptUtil');
```

Comment:
I've realized something is different with made by Tizen Studio.

In webide-common-tizentv,
First, there is no parameter for Certificate profile name, while there is required in Tizen Studio. (I guess this value is a real name for profile which can be activated.)

For test, I put these same values for making a Tizen certification.
"certificate profile name" : certificateprofilename
"keyfilename": keyfilename
"authorname": authorname
"password": password

And then, the results were different.

`In profiles.xml made by Tizen Studio.`
<profiles active="certificateprofilename" version="3.1">
<profile name="certificateprofilename">
<profileitem ca="C:\tizen-studio\tools\certificate-generator\certificates\developer\tizen-developer-ca.cer" distributor="0" key="C:/tizen-studio-data/keystore/author/keyfilename.p12" password="C:\tizen-studio-data\keystore\author\keyfilename.pwd" rootca=""/>
<profileitem ca="C:\tizen-studio\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-ca.cer" distributor="1" key="C:\tizen-studio\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-signer.p12" password="C:\tizen-studio-data\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-signer.pwd" rootca=""/>
<profileitem ca="" distributor="2" key="" password="" rootca=""/>
</profile>

`In profiles.xml made by webide-common-tizentv`
<profiles active="authorname" version="3.1">
  <profile name="authorname">
    <profileitem ca="E:\dev\Wits\node_modules\@tizentv\webide-common-tizentv\tools\certificate-generator\certificates\developer\tizen-developer-ca.cer" distributor="0" key="E:\dev\Wits\resource\Author\keyfilename.p12" password="E:\dev\Wits\resource\Author\keyfilename.pwd" rootca=""/>
    <profileitem ca="E:\dev\Wits\node_modules\@tizentv\webide-common-tizentv\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-ca.cer" distributor="1" key="E:\dev\Wits\node_modules\@tizentv\webide-common-tizentv\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-signer.p12" password="E:\dev\Wits\node_modules\@tizentv\webide-common-tizentv\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-signer.pwd" rootca=""/>
    <profileitem ca="" distributor="2" key="" password="" rootca=""/>
  </profile>


The results should be same.
Please check this point.